### ISSUE TITLE
Exception thrown when holding ctrl and simultaneously scroll the wheel on the mouse fast enough

### ISSUE THREAD

**Issue by tokarchyn** [pmndrs/use-gesture#551]
**Describe the bug**
Exception thrown when holding `ctrl` and simultaneously scroll the wheel on the mouse fast enough. 

```
TypeError
Cannot read properties of undefined (reading 'clientX')
    at distanceAngle (https://7rfcn.csb.app/node_modules/
use-gesture/core/dist/actions-71ad3053.esm.js:242:17
    at PinchEngine.pointerMove (https://7rfcn.csb.app/node_modules/
use-gesture/core/dist/actions-71ad3053.esm.js:1157:21
    at HTMLDivElement.eval (https://7rfcn.csb.app/node_modules/
use-gesture/core/dist/actions-71ad3053.esm.js:316:19
```

**Sandbox or Video**
Card-zoom sandbox from the documentation: https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/card-zoom

**Information:**
 - Use Gesture version: 10.2.21
 - Device: Laptop
 - OS: Windows 11
 - Browser Chrome

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [x] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

**Comment by LSafer** [comment]
Same with Firefox with the exception `P2 is undefined` (also reading `clientX`)

**Comment by dbismut** [comment]
I'm not able to reproduce... Are both your machines running windows? Are you using a trackpad? @LSafer @tokarchyn

**Comment by LSafer** [comment]
I think it got something to do with tracking pointerIds 

My setup was: Next.js + useSpring + useGuesture + mouse and a touch screen
with the ref passed in the options and not using the returned binding function and with onDrag and onPinch being setup

When I tracked touches count on `onDrag` it says 1, when I use one finger or the mouse.
But after a while (not sure why), it starts saying 2. Even though, I am still using one finger.
It is not the reason for the error. But, I think it is another side effect of the same problem

That's all I got since I got tiered and used [`react-zoom-pan-pinch`](https://github.com/prc5/react-zoom-pan-pinch)

**Comment by dbismut** [comment]
Tracking pointer ids can be a complex problem. Would you mind sharing a sandbox so I can have a look? Also are you using a Microsoft Surface by any chance?

**Comment by LSafer** [comment]
@dbismut Ok, it might take a while though and no I use `second screen` feature in my galaxy tab to test touch input (I use a windows desktop PC with keyboard and mouse)

**Comment by LSafer** [comment]
Sorry I couldn't reproduce it. I haven't committed the code until I changed it to a working solution

The code was using `react-image-lightbox` and I was searching for another solution since it doesn't work on react 18

**Comment by LSafer** [comment]
Umm, actually, the error occurs one this demo page:

https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/card-zoom

Click `Open in a new window` and zoom using `CTRL` + `mouse wheel`

**Comment by dbismut** [comment]
@LSafer thank you very much I'll have to get some hardware to reproduce this. Just so you know, pointer ids bug exists because I wanted the lib to support multi hands gestures, so two hands being able to interact pinching two different objects at the same time. This requires some logic that's a bit hard to master cross-devices...

**Comment by lindabeslist** [comment]
I have the same error on windows11 in chrome in the sandbox on desktop. You can reproduce it to drag the cart a bit to the right and zoom in and out with ctrl key and scrolling with the mousewheel.
![image](https://user-images.githubusercontent.com/7568777/202899328-55a6ce00-93d1-4b1b-9f0e-9839f78d0fd0.png)

**Comment by baptisteArno** [comment]
I have lots of users who get this error on Windows! Any chance we can suppress this error just by null checking where it reads clientX without having to reproduce the issue? @dbismut

**Comment by dbismut** [comment]
I guess we could indeed...

**Comment by dbismut** [comment]
Hi @baptisteArno and everyone. I've attempted a blind fix here #596 
Any way you could test if it fixes anything?

**Comment by baptisteArno** [comment]
Can it be pulled from npm repo?

**Comment by baptisteArno** [comment]
Unfortunately, I don't have a way to check if it fixes the issue, as I can only "test" this in production, and I'd rather not to…

@lindabeslist Can you try it out?

**Comment by tokarchyn** [comment]
@dbismut The error is gone 🙂I can easily reproduce an error on all previous versions, but not on `pmndrs/use-gesture#pull/596/head`.

**Comment by dbismut** [comment]
Great! Thanks for testing. I'll release this asap.

**Comment by lindabeslist** [comment]
Indeed, the error is gone. Thank you for fixing :)

**Comment by dbismut** [comment]
Released 10.2.26!
